AUTHORS

```diff
@@ -0,0 +1,15 @@
+# This is the official list of HTSJDK-Next authors for copyright purposes.
+#
+# The AUTHORS file lists the copyright holders.
+# For example, Broad Institute employees are not listed here, 
+# because Broad Institute holds the copyright.
+
+# This file should be updated whenever a new contributor merges their first commit.
+
+# Names should be added to this file as
```

Comment:
Should we separate the sections for organization/individuals? I guess that it might be easier for maintenance and easily pinpoint authors...

Reply:
I've split them up more clearly